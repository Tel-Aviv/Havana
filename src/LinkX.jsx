// @flow
import React from 'react';
import { Layout, Icon,  } from 'antd';

import { Tabs, Dropdown, Menu, message  } from 'antd-rtl';
const { TabPane } = Tabs;

const LinkX = () => {
    return <div>link
            </div>
}

export default LinkX;