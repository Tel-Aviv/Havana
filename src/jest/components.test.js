import React from 'react';
import renderer from 'react-test-renderer';

import LinkX from '../LinkX';

test('sss', () => {
    const component = renderer.create(
        <LinkX />
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})