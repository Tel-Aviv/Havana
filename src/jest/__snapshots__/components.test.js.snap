// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sss 1`] = `
<div>
  link
</div>
`;
